### PR TITLE
Added Manual Scripts

### PR DESCRIPTION
**Pull request by kowsisoundhar12**
Added New Scripts:
'Test_TC_LO_2_1',
'Test_TC_LO_3_1',
'Test_TC_DESC_2_1',
'Test_TC_DESC_2_2'
'Test_TC_DIAGLOG_1_1',
'Test_TC_DIAGLOG_2_1',
'Test_TC_DIAGLOG_2_2',
'Test_TC_DIAGLOG_3_1'

Updated Following Scripts:
'Test_TC_MF_1_1',
'Test_TC_MF_1_2',
'Test_TC_MF_1_7',
'Test_TC_MF_1_11',
'Test_TC_MF_1_12',
'Test_TC_MF_1_13',
'Test_TC_MF_1_16',
'Test_TC_MF_1_25',
'Test_TC_MF_1_26',
'Test_TC_MF_1_27',
'Test_TC_MF_1_28',
'Test_TC_SC_4_8',
'Test_TC_SU_2_1',
'Test_TC_SU_2_2',
'Test_TC_SU_2_3',
'Test_TC_SU_2_5',
'Test_TC_SU_2_7',
'Test_TC_SU_3_2',
'Test_TC_SU_3_3',
'Test_TC_SU_3_4',
'Test_TC_SU_4_1',
'Test_TC_SU_4_2'

Remove following manual script:
'Test_TC_BDX_1_1',
'Test_TC_BDX_1_3',
'Test_TC_BDX_1_5',
'Test_TC_BDX_1_6',
'Test_TC_BDX_2_3',
'Test_TC_BDX_2_4',
'Test_TC_BDX_2_5'

### CODE REVIEW

**Review by bzbarsky-apple — APPROVED** [review]

File: src/app/tests/suites/tests.js [hunk 1]
```diff
@@ -216,8 +212,6 @@ function getManualTests() {
     'Test_TC_MF_1_24',
     'Test_TC_MF_1_25',
     'Test_TC_MF_1_26',
-    'Test_TC_MF_1_27',
```
Comment (krypton36) [inline src/app/tests/suites/tests.js:219]: Is there a reason why these are being removed? There were just added in this [PR](https://github.com/project-chip/connectedhomeip/pull/17837)

**Review by krypton36 — CHANGES_REQUESTED** [review]

File: src/app/tests/suites/tests.js (hunk 1 above)
Comment (raju-apple) [reply src/app/tests/suites/tests.js:219]: Might be a case of TC name being renamed ? @kowsisoundhar12 can you confirm ?

Comment (kowsisoundhar12) [reply src/app/tests/suites/tests.js:219]: @raju-apple @krypton36 recently i updated this scripts based on latest  xl sheet but Manjunath PR is conflict with this PR so i removed this manual script

https://docs.google.com/spreadsheets/d/1h4d4J90KAxFqUL_cXZHyM3pHjHCEXzFITkQhN2KMqDY/edit#gid=710954404

**Review by krypton36 — APPROVED** [review]

**Comment by bzbarsky-apple** [comment]
Fast-tracking test additions.

**Pull request merged**